CRM/Case/Form/Task.php

```diff
@@ -76,7 +76,7 @@ protected function getRows(): array {
       if (empty($caseID) && !empty($this->_caseIds[$index])) {
         $caseID = $this->_caseIds[$index];
       }
-      $rows[] = ['contactId' => $contactID, 'caseId' => $caseID];
+      $rows[] = ['contact_id' => $contactID, 'schema' => ['caseId' => $caseID, 'contactId' => $contactID]];
```

Comment:
I haven't confirmed it's this line but I think something in this PR breaks sending a case email to someone who is not the client. It can no longer resolve e.g. `{case.id}`